azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/nav.vm

```diff
@@ -32,13 +32,33 @@
     color: red;
     cursor: pointer;
   }
+
+  #close {
+    float: right;
+    display: inline-block;
+    padding: 2px 5px;
+    background: #ccc;
+    color: #fff;
+  }
 </style>
 <script type="text/javascript">
   function navMenuClick(url) {
     window.location.href = url;
   }
 
+  function dismissBanner() {
+    var dismissBanner = sessionStorage.getItem("dismissBanner");
```

Comment:
Have we considerred using `localStorage`? What's disadvantages of it over `sessionStorage`?

Reply:
key stored in localStorage never expires until explicitly cleaning it. SessionStorage will last as long as the browser tab persists.

Reply:
I see clearnning is enforced when button is submitted or clearred. So what's disadvantages of persisting the data across all tabs?

Reply:
i see your point. After thinking, localstorage could be a better option in some sense.